Refactor/xacro launch generic elite description - generic elite arm robot description

AdarshKaran opened the pull request:
Added use_gazebo and use_arm_type arguments,  changed urdf to give flexibility to use other elite arms by just adding meshes and config files, renamed filed

AdarshKaran committed:
created elite_gazebo package,Combined urdf, added xacro, macro and config files, Refactored launch arguments-added use_gazebo arg in simulate_ec66.launch.py, restructured the packages

AdarshKaran committed:
Added elite_gazebo package, Refactored all launch files, restructured the packages structure, modified urdf files to xacro using macros and yaml config files

AdarshKaran committed:
Updated xacro name in elite_moveit package

AdarshKaran committed:
Added use_gazebo and use_arm_type arguments,  changed urdf to give flexibility to use other elite arms by just adding meshes and config files, renamed filed

padhupradheep commented on elite_moveit/config/ec66_simulation.urdf.xacro:
```diff
@@ -2,13 +2,13 @@
-    <xacro:include filename="$(find elite_description)/urdf/ec66_description.urdf" />
-
+    <!-- Import ec66 urdf file with use_gazebo set to true -->
+    <xacro:include filename="$(find elite_description)/urdf/ec66_description.urdf.xacro">
```
I think you need to update this.

padhupradheep reviewed (changes requested):
Overall LGTM! 

Could you please make a short video including the execution of MoveIt packages?

AdarshKaran committed:
1. reorganized and changed file names
2. made the elite arm xacro modular and compatible for both ec66 and cs66(yet to be added)
3. added initial positions for arm

AdarshKaran commented:
i have made the updates to the elite packages anna. kindly let me know if it is okay. I will then add the functionality for cs66 arm too

padhupradheep commented:
But there is still an issue with the MoveIt package right?

AdarshKaran commented:
yes anna. But, the description files were updated.

AdarshKaran committed:
Updated package.xml, fixes in launch files and moveit package

padhupradheep commented:
Is this PR still relevant?

AdarshKaran commented:
No Pradheep. the migration and refactoring has been addressed in pr #64 . this can be closed without merging
